test/e2e/framework/operator_util.go

```diff
@@ -61,6 +61,7 @@ func NewRedisCluster(name, namespace, tag string, nbMaster, replication int32) *
 								"--t=10s",
 								"--d=10s",
 								"--ns=$(POD_NAMESPACE)",
+								"--ns=$(POD_IP)",
```

Comment:
Looks like a copy/paste error:
```suggestion
								"--ip=$(POD_IP)",
```

Reply:
good catch